KDCS.js

```diff
@@ -34,28 +37,57 @@ const steps = {
 <br>
         Get pork gems and autosell<br>
         Visit meatsmith & accept quest<br>
-        Visit Toot Oriole & get pork gems<br>
         Do Clan consults<br>
         Then run 'kdcs postAscension'
         `);
     },
 
     postAscension: function() {
+        
+        // vote at the voting booth
         visitUrl("place.php?whichplace=town_right&action=townright_vote");
         visitUrl("choice.php?option=1&whichchoice=1331&g=2&local%5B%5D=2&local%5B%5D=3");
         visitUrl("place.php?whichplace=town_right&action=townright_vote");
-
+        
+        //get consults 
+        cliExecute("fortune cwbot pizza batman thick");
+        wait(5);
+        cliExecute("fortune cwbot pizza batman thick");
+         wait(5);
+        cliExecute("fortune cwbot pizza batman thick");
+        
+        
+        // accept meatsmith quest
+        visitUrl('shop.php?whichshop=meatsmith&action=talk');
+        runChoice(1);
+
+        // visit the toot oriole
+        visitUrl("tutorial.php?action=toot");
         visitUrl("tutorial.php?action=toot");
 
         cliExecute("use 1 letter from King Ralph XI");
         cliExecute("use 1 pork elf goodies sack");
-
-        cliExecute("pull 1 snow suit");
+        
+        /* 
+        
+        // Need to figure out a way to count how many of each gem I have and sell them, do I need to pull code for $item from libram?
+        
+        autosell(availableAmount($item`hamethyst`);
+        autosell(availableAmount($item`baconstone`);
+        
+        // keep 2 porqouises if we have more
+        autosell(availableAmount($item`porquoise`)-2, $item`porqouise`);
```

Comment:
You'll want to be careful with this math. I believe calling autosell with 0 actually means sell all my things.

Reply:
ah yes, maybe I can just closet the two porquoises beforehand instead of doing this math? 

Also i just realised that I don't actually define _what_ i'm selling with the baconstone and hamethysts!